frontend/src/components/common/LogViewer.tsx

```diff
@@ -99,10 +99,12 @@ export function LogViewer(props: LogViewerProps) {
   };
 
   function downloadLog() {
+    const time = new Date().toISOString().replace(/:/g, '-').slice(0, 19);
```

Comment:
It's more readable to use:
```typescript
.slice(0, -5)
```

And add a comment, like "Discard the last 5 digits as they we just want the time till the seconds".